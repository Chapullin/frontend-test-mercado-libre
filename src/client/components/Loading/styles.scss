.loading {
  width: 120px;
  display: inline-block;
  margin: auto;
  display: inherit;
}