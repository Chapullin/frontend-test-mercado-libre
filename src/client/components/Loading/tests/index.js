import React from "react";
import renderer from "react-test-renderer";
import Loading from "../../Loading";
import { BrowserRouter as Router, Route, Link, Switch } from "react-router-dom";

describe("Loading", () => {
  it("should match snapshot - base", () => {
    const component = renderer
      .create(
        <Loading />
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
