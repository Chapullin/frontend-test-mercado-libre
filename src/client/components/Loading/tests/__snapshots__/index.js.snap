// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match snapshot - base 1`] = `
<svg
  className="loading"
  height="100%"
  preserveAspectRatio="xMidYMid"
  viewBox="0 0 100 100"
  width="100%"
  xmlns="http://www.w3.org/2000/svg"
>
  <g
    transform="rotate(0 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.9166666666666666s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(30 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.8333333333333334s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(60 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.75s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(90 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.6666666666666666s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(120 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.5833333333333334s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(150 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.5s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(180 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.4166666666666667s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(210 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.3333333333333333s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(240 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.25s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(270 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.16666666666666666s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(300 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="-0.08333333333333333s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
  <g
    transform="rotate(330 50 50)"
  >
    <rect
      fill="#3483fa"
      height="12"
      rx="9.4"
      ry="4.8"
      width="6"
      x="47"
      y="24"
    >
      <animate
        attributeName="opacity"
        begin="0s"
        dur="1s"
        keyTimes="0;1"
        repeatCount="indefinite"
        values="1;0"
      />
    </rect>
  </g>
</svg>
`;
